dist/tools/compile_and_test_for_board/compile_and_test_for_board.py

```diff
@@ -52,9 +52,11 @@
   --applications APPLICATIONS
                         List of applications to test, overwrites default
                         configuration of testing all applications
+                        Applications can contain wildcards.
```

Comment:
It should also say that the wildcard will be evaluated as relative to the RIOT directory somehow.